### ISSUE TITLE
Listbox: Should collapsible listboxes require the aria-expanded state?

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria-practices#596]
The entry for listbox does not seem to require aria-expanded to be set, should it?

Note, in case it's relevant, that comboboxes are required to set aria-expanded=true when its listbox is visible.

**Comment by mcking65** [comment]
@0ddfell0w, the [collapsible listbox example](http://w3c.github.io/aria-practices/examples/listbox/listbox-collapsible.html)
has `aria-expanded="true"` when the listbox is visible. 
Similar to the menu button pattern, though, it does not have the aria-expanded attribute when it is collapsed.

This is a forward looking example as it uses the new ARIA 1.1 token of `aria-haspopup="listbox"`, which is currently treated by assistive technologies in the same as `aria-haspopup="true"`. That means that screen readers read it as a menubutton when it is in its collapsed state. While that is not a show stopper, it is not ideal. At least it communicates that the button will show something like a menu if activated.

**Comment by mcking65** [comment]
@0ddfell0w, I'm pretty sure that your question has been answered. So, closing.

That said, I'm not completely sure what you were referencing when you said "The entry for listbox ". I am assuming you meant the [listbox pattern in the APG](http://w3c.github.io/aria-practices/#Listbox).

The listbox design pattern does not specifically describe the collapsable listbox. That's because it is a combination of button and listbox. We could have made a separate pattern for collapsable listbox like we have for menubutton. However, one reason for this example was to drive discussion for how HTML `select@size=1` should be mnappend and rendered (See w3c/html-aam#46). In APG 1.2, I expect that we will modify the listbox pattern to describe use of aria-expanded and we will then also have a new version of the collapsable listbox example that is better than this one. Because of that future direction, I don't think it is best to have a separate collapsable listbox pattern at this time.

If you'd like to have further discussion, please feel free to re-open this issue.
